modules/EvseManager/manifest.yaml

```diff
@@ -139,6 +139,15 @@ config:
       Set to true to enable dirty workaround for some IMD hardware.
     type: boolean
     default: false
+  auth_mode:
+    description: >-
+      Specifies the authorization mode. Either PlugAndCharge, AutoCharge or Disabled
+    type: string
+    enum:
+      - Autocharge
+      - PnC
+      - Disabled
+    default: PnC
```

Comment:
Disabled would mean RFID/EIM but RFID/EIM could also apply when the mode is PnC or Autocharge, depending on the mechanism that the EVSE provides / the user chooses. So a better name would propably be "pnc_mode" or "ev_auth_mode". Default should be Disabled, thats correct. 

We actually also thought about removing this configuration completely to let the EvseManager just use whats there, but Autocharge was conflicting with PnC because Autocharge happens "before" PnC. We should discuss this again.

Reply:
Removed